app/actions/sendModalActions.js

```diff
@@ -7,9 +7,10 @@ export const ID = 'send'
 
 export const getRecipientData = createActions(
   ID,
-  ({ url, chain }) => (): RecipientData => {
+  ({ url, chain, net }) => async (): Promise<RecipientData> => {
     try {
-      if (chain === 'neo2') return parseQRCode(url)
+      if (chain === 'neo2' || url.includes('neo:'))
+        return parseQRCode({ url, net, chain })
```

Comment:
We will attempt to parse the nep9 QR code if it includes protocol otherwise we will just assume it is an address